developer/src/kmcmplib/src/NamedCodeConstants.cpp

```diff
@@ -117,85 +114,61 @@ char *kmc_strupr(char *s) {
   return s;
 }
 
-KMX_BOOL NamedCodeConstants::IntLoadFile(const KMX_CHAR *filename)
-{
+KMX_BOOL NamedCodeConstants::LoadFile(PFILE_KEYBOARD fk, const KMX_WCHAR *filename) {
   const int str_size = 256;
-  FILE *fp = NULL;
 
   if (CheckFilenameConsistency(filename, FALSE) != 0) {
     return FALSE;
   }
 
-  fp = Open_File(filename, "rt");
-  if(fp == NULL) {
-    return FALSE;  // I3481
+  auto szNameUtf8 = string_from_u16string(filename);
+
+  int FileSize;
+  KMX_BYTE* Buf;
+  if(!loadfileproc(szNameUtf8.c_str(), fk->extra->kmnFilename.c_str(), nullptr, &FileSize, msgprocContext)) {
```

Comment:
I'm having trouble tracing how `loadfileproc` has been declared for use in this file.  I don't doubt that it _is_ declared... but not being able to trace that is disturbing me a bit.

I suppose it was already set up in a prior PR, which would've made changes elsewhere at the time... but not here?

Reply:
Declaration is in the kmcmplib.h header. It's a global (as is `msgproc`). Globals may be refactored one day.

https://github.com/keymanapp/keyman/blob/77da56e7487ee70d8acdee4492362a7ab248e149/developer/src/kmcmplib/src/kmcmplib.h#L26